src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -8,6 +8,9 @@ namespace Microsoft.Maui
 {
 	public static class ViewExtensions
 	{
+		const int DefaultAutomationTagId = -1;
+		public static int AutomationTagId { get; set; } = DefaultAutomationTagId;
```

Comment:
Is this the best place for this? Where is the final place "planned" Is this a context thing or a app builder thing? Maybe it can be here but set via another way?

Reply:
It's an appium thing

It's the TagID that we'll use to locate stuff for Appium 

@Redth any thoughts about this one?

Reply:
I mean the location of the `AutomationTagId` property. Will the user ever want to change this?

Reply:
I see the https://github.com/dotnet/maui/pull/491/commits/91fc3c8cd8b8cb01cb76ee38e73a7df751c0dced commit came in after I started the review.